packages/react/src/components/SkeletonIcon/SkeletonIcon.tsx

```diff
@@ -4,18 +4,29 @@
  * This source code is licensed under the Apache-2.0 license found in the
  * LICENSE file in the root directory of this source tree.
  */
-
 import PropTypes from 'prop-types';
 import React from 'react';
 import classNames from 'classnames';
 import { usePrefix } from '../../internal/usePrefix';
 
-const SkeletonIcon = ({ className, ...other }) => {
+interface SkeletonIconProps {
+  /**
+   * Specify an optional className to add.
+   */
+  className?: string;
+
+  /**
+   * The CSS styles.
+   */
+  style?: React.CSSProperties;
+}
+
+const SkeletonIcon: React.FC<SkeletonIconProps> = ({ className, ...other }) => {
   const prefix = usePrefix();
 
   const skeletonIconClasses = classNames({
     [`${prefix}--icon--skeleton`]: true,
-    [className]: className,
+    [className!]: className,
```

Comment:
We can instead change this to something like this:

```jsx
  const skeletonIconClasses = classNames(className, {
    [`${prefix}--icon--skeleton`]: true,
  });
```

Reply:
Okay, thank you for reviewing 🙂